VirtoCommerce.Platform.Web/Modularity/ManifestModuleCatalog.cs

```diff
@@ -170,25 +170,50 @@ public override void Validate()
             return result;
         }
 
+        private static void CopyLocalizationAssemblies(string sourceDirectoryPath, string targetDirectoryPath)
+        {
+            var cultureNames = ConfigurationHelper.SplitAppSettingsStringValue("VirtoCommerce:Resources:CultureNames");
+
+            foreach (var languageCode in cultureNames)
+            {
+                if (!string.IsNullOrEmpty(languageCode))
+                {
+                    var separator = Path.DirectorySeparatorChar;
+                    var localizationDirectoryPath = Path.Combine(sourceDirectoryPath, $"{languageCode}{separator}");
+                    var localizationTargetDirectoryPath = Path.Combine(targetDirectoryPath, $"{languageCode}{separator}");
+
+                    CopyAssemblyFiles(localizationDirectoryPath, localizationTargetDirectoryPath);
+                }
+            }
+        }
+
         private static void CopyAssemblies(string sourceParentPath, string targetDirectoryPath)
         {
             if (sourceParentPath != null)
             {
-                var sourceDirectoryPath = Path.Combine(sourceParentPath, "bin\\");
+                var separator = Path.DirectorySeparatorChar;
+                var sourceDirectoryPath = Path.Combine(sourceParentPath, $"bin{separator}");
 
-                if (Directory.Exists(sourceDirectoryPath))
-                {
-                    var sourceDirectoryUri = new Uri(sourceDirectoryPath);
+                CopyLocalizationAssemblies(sourceDirectoryPath, targetDirectoryPath);
+
+                CopyAssemblyFiles(sourceDirectoryPath, targetDirectoryPath);
+            }
+        }
 
-                    foreach (var sourceFilePath in Directory.EnumerateFiles(sourceDirectoryPath))
+        private static void CopyAssemblyFiles(string sourceDirectoryPath, string targetDirectoryPath)
+        {
+            if (Directory.Exists(sourceDirectoryPath))
```

Comment:
try to invert this, probably your code will be shorter without extra braces

Reply:
I agree with you. But we use a single return code convention here.